storage-proofs/src/parameter_cache.rs

```diff
@@ -182,31 +181,14 @@ where
     }
 
     fn get_groth_params<R: RngCore>(
-        rng: Option<&mut R>,
-        circuit: C,
+        _rng: Option<&mut R>,
+        _circuit: C,
```

Comment:
can you remove these argumentes, no need to keep them around

Reply:
Hrm, so sort of.  I tried removing both at once and all hell broke loose not knowing about the circuit in other parts, so I reverted for the moment and just removed the rng.